interfaces/mir100-sse/status.txt

```diff
@@ -0,0 +1,43 @@
+{ allowed_methods: null,
+  battery_percentage: 85.5,
+  battery_time_remaining: 82092,
+  distance_to_next_target: 0,
+  errors: [],
+  footprint: '[[0.506,-0.32],[0.506,0.32],[-0.454,0.32],[-0.454,-0.32]]',
+  map_id: 'b1bf0757-1e42-11e9-8200-94c6911e8fa3',
+  mission_queue_id: null,
+  mission_queue_url: null,
+  mission_text: 'Starting...',
+  mode_id: 7,
+  mode_text: 'Mission',
+  moved: 21980.45,
+  position:
+   { orientation: -76.06780242919922,
+     x: 57.908668518066406,
+     y: 5.036342144012451 },
+  robot_model: 'MiR100',
+  robot_name: 'FRIDA',
+  serial_number: '180100002100871',
```

Comment:
You might want to delete this file, not sure if someone can do something bad with our serial number